fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStat.java

```diff
@@ -17,22 +17,11 @@
 
 package org.apache.doris.statistics;
```

Comment:
it is better to remove ColumnStat totally, since it is almost replaced by ColumnStatistic.

Reply:
I will do this in another PR, since this would infect many files